setup.cfg

```diff
@@ -0,0 +1,63 @@
+[metadata]
+name = wxflow
+version = attr: wxflow.__version__
+description = Tools for Weather Workflows
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = NOAA/NWS/NCEP/EMC
+#author_email = first.last@domain.tld
+keywords = NOAA, NWS, NCEP, EMC
+home_page = https://github.com/noaa-emc/wxflow
+license = GNU Lesser General Public License v3 or later (LGPLv3+)
+classifiers =
+  Development Status :: 4 - Beta
+  Intended Audience :: Developers
+  Intended Audience :: Science/Research
+  License :: OSI Approved :: GNU Lesser General Public License
+  Natural Language :: English
+  Operating System :: OS Independent
+  Programming Language :: Python
+  Programming Language :: Python :: 3
```

Comment:
If you specify `Python :: 3` here, why do you need to include 3.7 - 3.11? My understanding was that `Python :: 3`` should cover all Python version 3 releases. Is this just decorative?

Reply:
Because the underlying code will not work w/ 3.6 or below because the dependencies have become deprecated.
I urge you to try  and install this w/ e.g. 3.5

Reply:
I believe you. I was just curious as to what your reasoning was here. I see you addressed this [here](https://github.com/NOAA-EMC/wxflow/pull/3/files#diff-dee67b273be5446ee34cb0d12bb048fe4825d67bb1e4a1a80610c72abd3ff9a0R10). Maybe this is OK.